clients/nodejs/github.com/anchovycation/metronom.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "metronom",
+    "description": "User friendly Redis ORM for Node.js with asynchronous and TypeScript support.",
+    "homepage": "https://anchovycation.github.io/metronom/",
+    "recommended": true,
```

Comment:
This attribute is reserved for the community.

```suggestion
```

Reply:
@itamarhaber  I made the correction you requested, thank you for your interest